actions.md

```diff
@@ -143,7 +143,7 @@ These are the primary actions that a user of libaktualizr can perform through th
       - [x] Set new version to pending status after an OSTree update trigger (aktualizr_test.cc)
       - [x] Send EcuInstallationAppliedReport to server after an OSTree update trigger (aktualizr_test.cc)
       - [x] Uptane check for updates and manifest sends are disabled while an installation is pending reboot (aktualizr_test.cc)
-      - [ ] Trigger a system reboot if `reboot_after_install` is set in the configuration
+      - [ ] Trigger a system reboot if `force_install_completion` is set in the package manager configuration. Emulates a reboot in case of the fake package manager.
```

Comment:
I would put two entries, one which explicitly mentions OSTree, and the other the fake package manager.

It should also be possible to write a test in `aktualizr_test.cc` which uses the fake package manager implementation and then check this box.

Reply:
ok